src/external_app/external_messaging.ts

```diff
@@ -180,6 +207,7 @@ export interface ExternalConfig {
   canCommissionMatter: boolean;
   canImportThreadCredentials: boolean;
   hasAssist: boolean;
+  hasQRScanner: boolean;
```

Comment:
For future proofing, should we make this an integer to indicate version number ?

Reply:
Might be wise, I can see different type of codes needed to be scanned

Reply:
```suggestion
  hasQRScanner: number;
```